### PR TITLE
V2.0.0 final release

### PR DESCRIPTION
**Pull request by uk-bolly**
**Overall Review of Changes:**
Issues addressed
typos fixed

**Issue Fixes:**
#21 thanks to @chrisdleech-pol
#22 thanks to @mfortin

**Enhancements:**
typo fixed

**How has this been tested?:**
Manually

### CODE REVIEW

File: tasks/section_1/cis_1.1.x.yml [hunk 1]
```diff
@@ -1,51 +1,84 @@
...
       name: /tmp
       src: "{{ item.device }}"
       fstype: "{{ item.fstype }}"
       state: present
-      opts: defaults,{% if amazon2cis_rule_1_1_4 %}nodev,{% endif %}{% if amazon2cis_rule_1_1_3 %}noexec,{% endif %}{% if amazon2cis_rule_1_1_5 %}nosuid{% endif %}
-  notify: systemd restart tmp.mount
-  with_items:
-      - "{{ ansible_mounts }}"
+      opts: defaults,{% if amazon2cis_rule_1_1_3 %}noexec,{% endif %}{% if amazon2cis_rule_1_1_4 %}nodev,{% endif %}{% if amazon2cis_rule_1_1_5 %}nosuid{% endif %}
+  notify: Remount_tmp
```
Comment (MrSteve81) [inline tasks/section_1/cis_1.1.x.yml:37]: Remount_tmp is not in handlers but the old systems restart tmp.mount is still there

**Review by MrSteve81 — CHANGES_REQUESTED** [review]
Check notes sent

**Review by MrSteve81 — APPROVED** [review]
Approved

File: tasks/section_1/cis_1.1.x.yml (hunk 1 above)
Comment (uk-bolly) [reply tasks/section_1/cis_1.1.x.yml]: addressed in ne update

**Pull request merged**